README.md

```diff
@@ -19,141 +20,189 @@ When the reader has completed this Code Pattern, they will understand how to:
+  ![Choose Python](doc/source/images/openshift-choose-python.png)
 
-* Follow the instructions for `ACTION: Get data_mart_id (GUID) and apikey` using the [IBM Cloud CLI](https://cloud.ibm.com/docs/cli/index.html#overview)
+* Give your app a name and add `https://github.com/IBM//monitor-custom-ml-engine-with-watson-openscale` for the github repo, then click `Create`.
 
-Get an IAM apikey:
-```
-ibmcloud login --sso
-ibmcloud iam api-key-create 'my_key'
-```
+  ![Add github repo](https://raw.githubusercontent.com/IBM/pattern-utils/master/openshift/openshift-add-github-repo.png)
 
-Get Watson OpenScale instance GUID:
-```
-ibmcloud resource service-instance <Watson_OpenScale_instance_name>
-```
+### 2. Create the config map
 
-* Enter the `GUID` as the `instance_guid` and the iam `API Key` as the `apikey` in the next cell for the `WATSON_OS_CREDENTIALS`.
-* In the cell after `ACTION: Add your PostgreSQL credentials here` enter the credentials from the [Databases for PostgreSQL DB](https://cloud.ibm.com/catalog/services/databases-for-postgresql) that you created earlier.
+* Click on the `Resources` tab and choose `Config Maps` and then click the `Create Config Map` button.
+  * Provide a `Name` for the config map.
+  * Click `Add Item` and add a key named `URL` and under `Enter a value...`, enter auth url of Cloud Pak for Data instance under which the prediction model is deployed.
+  * Click `Add Item` and add a key named `USERNAME` and under `Enter a value...`, enter username to Cloud Pak for Data instance under which the prediction model is deployed.
+  * Click `Add Item` and add a key named `PASSWORD` and under `Enter a value...`, enter the password to Cloud Pak for Data instance under which the prediction model is deployed.
+  * Hit the `Create` button.
+  * Click on your new Config Map's name.
+  * Click the `Add to Application` button.
+  * Select your application from the pulldown.
+  * Click `Save`.
 
-* Move your cursor to each code cell and run the code in it. Read the comments for each cell to understand what the code is doing. **Important** when the code in a cell is still running, the label to the left changes to **In [\*]**:.
-  Do **not** continue to the next cell until the code is finished running.
+  ![config_map.png](doc/source/images/config_map.png)
 
-# Sample output
+* Go to the `Applications` tab, choose `Deployments` to view the status of your application.
 
-### GET the application server deployments
+## 3. Get a secure endpoint
 
-Navigate a browser to `http://<ip_address>:<port>/v1/deployments`
-or run the [test_deployments.py](test/deployments_api.py) script following the  [test/README.md](test/README.md) instructions.
+* From the OpenShift or OKD UI, under `Applications` ▷ `Routes` you will see your app.
+  * Click on the application `Name`.
+  * Under `TLS Settings`, click on `Edit`.
+  * Under `Security`, check the box for `Secure route`.
+  * Hit `Save`.
 
+## 4. Run the web app
```

Comment:
When this section is expanded, the numbering is very confusing. You will see #4, then #3, then #4 again. Maybe make this a separate page? Or somehow indent the expand/collapse section more.

Reply:
I agree. I've removed the numbers.